src/ansible_navigator/_yaml.py

```diff
@@ -62,8 +74,15 @@ def represent_scalar(self, tag, value, style=None):
         return super().represent_scalar(tag, value, style)
 
 
-# from http://stackoverflow.com/a/15423007/115478
-def _is_multiline_string(value):
+def _is_multiline_string(value: str):
+    """Determine if a string is multiline.
+
+    Note:
+    from http://stackoverflow.com/a/15423007/115478
+
+    :param value: The value to check
+    :return: a boolean indicating is the string is multiline
```

Comment:
Other docstrings have this capitalize. So it'd be consistent to do
```suggestion
    :return: A boolean indicating if the string is multiline
```
Also `s/is/if/`

Reply:
ty